packages/cli/src/util/dev/parse-query-string.ts

```diff
@@ -0,0 +1,47 @@
+export function parseQueryString(querystring?: string) {
+  let map = new Map<string, string[]>();
```

Comment:
nit: prefer const

```suggestion
  const map = new Map<string, string[]>();
```

Reply:
Fixed in [2d36b10](https://github.com/vercel/vercel/pull/8183/commits/2d36b101432ad91b96128c33547568c3fc1c8d0a). I guess we should update to https://github.com/vercel/style-guide 🙂